pkg/acsengine/artifacts.go

```diff
@@ -261,37 +261,55 @@ func kubernetesAddonSettingsInit(profile *api.Properties) []kubernetesAddonSetti
 	}
 }
 
-func kubernetesManifestSettingsInit(profile *api.Properties) []kubernetesFeatureSetting {
-	return []kubernetesFeatureSetting{
+func kubernetesManifestSettingsInit(profile *api.Properties) []kubernetesAddonSetting {
+	return []kubernetesAddonSetting{
```

Comment:
Let's just use `kubernetesFeatureSettings` instead. Delete the `kubernetesAddonSetting` struct and add the rawScript field to  `kubernetesFeatureSettings`.